tests/FieldAlias.php

```diff
@@ -87,6 +91,10 @@ public function testLegacyGroupBy()
     public function testLegacyGroupByFunction()
     {
         $mapper = test_spot_mapper('SpotTest\Entity\Legacy');
+        // Currently it works with mysql only
+        if (!$mapper->connectionIs('mysql')) {
+            $this->markTestSkipped('Not supported in Sqlite nor Postgres.');
```

Comment:
Spot supports more than just 3 databases. These messages should be specific to MySQL only if the feature is only supported by MySQL. Something like `This feature is only supported in MySQL`.